include/lsst/sphgeom/Region.h

```diff
@@ -149,9 +166,55 @@ class Region {
     static std::unique_ptr<Region> decode(std::uint8_t const * buffer, size_t n);
     ///@}
 
+    ///@{
+    /// `decodeBase64` deserializes a Region from an ASCII string produced by
+    /// encode and then base64-encoding that result.
+    ///
+    /// This method also interprets ':' as a delimiter for the elements of a
+    /// UnionRegion, to support cases where a union of region is constructed
+    /// server-side in a database as an concatenation with that delimiter.
```

Comment:
Would it be useful to add support for IntersectionRegion as well, via a different delimiter?

Reply:
I thought about that too, but I think it'd end up dead code.  And if I'm wrong, it might need to be more complex than allowing just one delimiter or the other (i.e. we'd need to work out associativity).